pyro/infer/trace_elbo.py

```diff
@@ -160,7 +160,7 @@ def loss_and_grads(self, model, guide, *args, **kwargs):
             # build a closure for loss_and_surrogate_loss
             weakself = weakref.ref(self)
 
-            @pyro.ops.jit.compile(nderivs=1)
+            @pyro.ops.jit.trace
```

Comment:
I don't think we should merge the jit changes into 0.4.0. I guess nobody is using the jit, but it seems wrong to put 0.4.1 jit code in a repo pinned to 0.4.0. I think you can mostly revert individual files.

Reply:
Makes sense. Let me revert that so that the other `0.4.1` branch only contains JIT changes and doesn't need to be merged anytime soon.